packages/middleware-body/src/indes.spec.ts

```diff
@@ -1,7 +1,7 @@
+import { HttpRequest, HttpResponse } from '@marblejs/core';
 import { of } from 'rxjs';
-import { HttpRequest, HttpResponse } from '../http.interface';
-import { Marbles } from '../util/marbles.spec-util';
-import { bodyParser$ } from './bodyParser.middleware';
+import { Marbles } from '../../core/src/util/marbles.spec-util';
+import { bodyParser$ } from './index';
```

Comment:
`import { bodyParser$ } from './';`

Reply:
In this case *index.ts* is more explicit